engine/src/test/java/com/redhat/cloud/notifications/templates/common/TestAdvisorTemplate.java

```diff
@@ -120,6 +119,7 @@ void testInstantEmailTitleForResolvedRecommendations() {
 
     @Test
     public void testInstantEmailTitleForNewRecommendations() {
+        eventTypeDisplayName = "New recommendation";
```

Comment:
This looks unused.

Reply:
Used by parent class [EmailTemplatesRendererHelper](https://github.com/RedHatInsights/notifications-backend/blob/master/engine/src/test/java/com/redhat/cloud/notifications/EmailTemplatesRendererHelper.java#L149).